tools/odrive/utils.py

```diff
@@ -31,6 +32,25 @@
 class OperationAbortedException(Exception):
     pass
 
+def set_motor_thermistor_coeffs(motor, Rload, R_25, T_25, Beta, Tmin, TMax):
+    temps = np.linspace(Tmin, Tmax, 1000)
+    tempsK = temps + 273.15
+
+    # https://en.wikipedia.org/wiki/Thermistor#B_or_%CE%B2_parameter_equation
+    r_inf = R_25 * np.exp(-Beta/T_25)
+    R_temps = r_inf * np.exp(Beta/tempsK)
+    V = Rload / (Rload + R_temps)
+
+    fit = np.polyfit(V, temps, 3)
+    p1 = np.poly1d(fit)
+    fit_temps = p1(V)
+
+    motor.motor_thermistor.poly_coefficients.idx_0 = float(fit_temps[0])
+    motor.motor_thermistor.poly_coefficients.idx_1 = float(fit_temps[1])
+    motor.motor_thermistor.poly_coefficients.idx_2 = float(fit_temps[2])
+    motor.motor_thermistor.poly_coefficients.idx_3 = float(fit_temps[3])
```

Comment:
I think the polynomial coefficients are in `fit` or `p1`, not in fit_temps.

Reply:
Either way, to avoid code duplication, we should have thermistors.py call this function (or other way around)